backend/server/routers/courses.py

```diff
@@ -280,7 +290,9 @@ def unselectCourse(userData: UserData, lockedCourses: list, unselectedCourse: st
                     "content": {
                         "application/json": {
                             "example": {
-                                "courses_unlocked_when_taken": ["COMP2511, COMP3311"]
+                                "courses_unlocked_when_taken": ["COMP2511, COMP3311"],
```

Comment:
this line is now irrelevant in the documentation since that's not included in the response

Reply:
Updated